spec/requests/discourse_code_review/repos_controller_spec.rb

```diff
@@ -125,9 +126,19 @@ def set_client(client)
         end
 
         it "says yes" do
-          get '/admin/plugins/code-review/organizations/org/repos/repo/has-configured-webhook.json'
+          get "/admin/plugins/code-review/organizations/org/repos/#{repo_name}/has-configured-webhook.json"
           expect(JSON.parse(response.body)).to eq('has_configured_webhook' => true)
         end
+
+        context "when repo name has a . in it" do
+          let(:repo_name) { "Some-coolrepo.org" }
+
+          it "does not 404 error" do
+            get "/admin/plugins/code-review/organizations/org/repos/#{repo_name}/has-configured-webhook.json"
+            expect(response.status).not_to eq(404)
```

Comment:
Instead of testing for the response code that we do not want. I think it is clear to test for the response code that we expect. Otherwise, this can be returning `500`s and the test will still pass.